website/src/components/Contributors/contributors.css

```diff
@@ -0,0 +1,88 @@
+.grid {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  flex-wrap: nowrap;
+  overflow: hidden;
+  place-items: center;
+}
+
+.grid-item {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  height: 100%;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: center;
+  min-width: 80vh;
+
+}
+
+.circle {
+  width: 35px;
+  height: 35px;
```

Comment:
It seems like the circles are not actually rendered like this:

<img width="225" alt="Screenshot 2025-05-01 at 5 57 52 PM" src="https://github.com/user-attachments/assets/011f4978-6800-4c3d-85ba-cb50a474159c" />

Let's make sure they have the same width and height